app/models/organisation.rb

```diff
@@ -0,0 +1,13 @@
+# == Schema Information
+#
+# Table name: organisation
+#
+#  id     :integer          not null, primary key
+#  name   :text
+#  org_id :text
+#
+
+class Organisation < ApplicationRecord
+  has_and_belongs_to_many :users
+  has_and_belongs_to_many :providers
```

Comment:
Odd that there is only one unit test added for the `User`. I'd add tests for these since they're one-liners.